Tasks/IISWebAppManagementOnMachineGroup/Utility.ps1

```diff
@@ -29,6 +29,11 @@ function Manage-IISWebSite
         [string] $appPoolIdentity,
         [string] $appPoolUsername,
         [string] $appPoolPassword,
+
+        [string] $configureAuthentication,
+        [string] $basicAuthentication,
```

Comment:
I was hoping there is only one authentication type (value set by a picklist) that we pass along and we don't have to manage multiple variables for each auth type.